service/controller/v2/resource/chart/update_test.go

```diff
@@ -34,7 +34,21 @@ func Test_Resource_Chart_newUpdateChange(t *testing.T) {
 			expectedUpdateState: nil,
 		},
 		{
-			description: "case 1: nonempty current state, different release version in desired state, expected desired state",
+			description: "case 1: nonempty current state, equal desired state, empty update change",
```

Comment:
Nice to modify/extend existing tests without explicitly testing `isChartModified`, only its external effect :+1: